src/main.ts

```diff
@@ -1,12 +1,68 @@
 import { Command, Helper } from './interfaces';
 import { Argv } from 'yargs';
+import createDir from './createDir';
+import renderFiles, { RenderFilesConfig } from './renderFiles';
+import npmInstall from './npmInstall';
+import typingsInstall from './typingsInstall';
+import { join } from 'path';
+import * as chalk from 'chalk';
+const pkgDir: any = require('pkg-dir');
+
+export interface CreateAppArgs extends Argv {
+	name: string;
+}
+
+const packagePath = pkgDir.sync(__dirname);
 
 const command: Command = {
 	description: 'Scaffolds a new command',
-	register(helper) {
+	register(helper: Helper) {
+		helper.yargs.option('n', {
+			alias: 'name',
+			describe: 'The App Name to create',
```

Comment:
The name of the application? just small the `The App Name to create` just reads funny to me.